core/services/chainlink/relayer_factory.go

```diff
@@ -67,6 +68,15 @@ func (r *RelayerFactory) NewEVM(config EVMFactoryConfig) (map[types.RelayID]evmr
 	}
 
 	if cmdName := env.EVMPlugin.Cmd.Get(); cmdName != "" {
+		if anyNotNil(config.GenChainStore,
+			config.GenEthClient,
+			config.GenLogBroadcaster,
+			config.GenLogPoller,
+			config.GenHeadTracker,
+			config.GenTxManager,
+			config.GenGasEstimator) {
+			return nil, errors.New("Gen* overrides are not supported in the loopp mode")
```

Comment:
Let's align the message and wrap like the others:
```suggestion
			return nil, errors.New("Gen* overrides are not available in LOOP Plugin mode: %w", stderrors.ErrUnsupported)
```

Reply:
Done.